apps/dashboard/src/hooks/use-create-workflow.ts

```diff
@@ -34,6 +40,13 @@ export function useCreateWorkflow({ onSuccess }: UseCreateWorkflowOptions = {})
 
       onSuccess?.();
     },
+
+    onMutate: () => {
+      showSavingToast(setToastId);
+    },
+    onError: (error) => {
+      showErrorToast(toastId, error);
```

Comment:
we should handle the same way errors when syncing between environments... 
on the api level we will be safe as it uses upsert usecase